AmplifyPlugins/Logging/Sources/AWSCloudWatchLoggingPlugin/Persistence/LogEntry.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import Foundation
+
+/// Represents an individual row in a log.
+struct LogEntry: Codable, Hashable {
+    
+    /// The timestamp representing the creation time of the log entry or event.
+    let created: Date
+
+    /// The logical tag of the log entry or event. This will likely be a
+    /// Category name.
+    let category: String
+    
+    let namespace: String?
```

Comment:
nit: can we have a comment explaining this property

Reply:
missing comment added